test failure #874

The following test appears to have failed:

[#874](https://circleci.com/gh/cockroachdb/cockroach/874):

```
=== RUN TestTypeMismatch
--- PASS: TestTypeMismatch (0.00s)
=== RUN TestSameKeyInserts
--- PASS: TestSameKeyInserts (0.00s)
=== RUN TestGroupCompactAfterTTL
--- FAIL: TestGroupCompactAfterTTL (0.00s)
	group_test.go:188: shouldn't be able to insert
	group_test.go:194: group.go:209: current group info &{Key:a.c Val:3 Timestamp:1427233625486030067 TTLStamp:1427233685486030067 Hops:0 NodeAddr:<nil> peerAddr:<nil> seq:0} newer than proposed info &{Key:a.c Val:3 Timestamp:1427233625486030067 TTLStamp:1427233685486030067 Hops:0 NodeAddr:<nil> peerAddr:<nil> seq:0}
=== RUN TestGroups100Keys
--- PASS: TestGroups100Keys (0.10s)
=== RUN TestSameKeySameTimestamp
--- PASS: TestSameKeySameTimestamp (0.00s)
=== RUN TestSameKeyDifferentHops
--- PASS: TestSameKeyDifferentHops (0.00s)
=== RUN TestGroupGetInfo
--- PASS: TestGroupGetInfo (0.00s)
--
--- PASS: TestConvergence (0.44s)
=== RUN TestGossipInfoStore
--- PASS: TestGossipInfoStore (0.00s)
=== RUN TestGossipGroupsInfoStore
--- PASS: TestGossipGroupsInfoStore (0.00s)
FAIL
FAIL	github.com/cockroachdb/cockroach/gossip	0.713s
?   	github.com/cockroachdb/cockroach/gossip/simulation	[no test files]
=== RUN TestGetFirstRangeDescriptor
I0324 21:47:09.861223     193 network.go:62] simulating gossip network with 3 nodes
I0324 21:47:09.866158     193 server.go:93] serving on /tmp/unix-socket166524144...
I0324 21:47:09.866265     193 server.go:93] serving on /tmp/unix-socket310313871...
I0324 21:47:09.866339     193 server.go:93] serving on /tmp/unix-socket558111650...
I0324 21:47:09.866481     193 gossip.go:373] bootstrapping gossip protocol using host /tmp/unix-socket558111650
I0324 21:47:09.866879     193 gossip.go:373] bootstrapping gossip protocol using host /tmp/unix-socket558111650
I0324 21:47:09.867179     193 gossip.go:373] bootstrapping gossip protocol using host /tmp/unix-socket166524144
I0324 21:47:09.873412     193 client.go:142] client /tmp/unix-socket558111650 connected
=== RUN TestTypeMismatch
--- PASS: TestTypeMismatch (0.00s)
=== RUN TestSameKeyInserts
--- PASS: TestSameKeyInserts (0.00s)
=== RUN TestGroupCompactAfterTTL
--- FAIL: TestGroupCompactAfterTTL (0.00s)
	group_test.go:188: shouldn't be able to insert
	group_test.go:194: group.go:209: current group info &{Key:a.c Val:3 Timestamp:1427233625486030067 TTLStamp:1427233685486030067 Hops:0 NodeAddr:<nil> peerAddr:<nil> seq:0} newer than proposed info &{Key:a.c Val:3 Timestamp:1427233625486030067 TTLStamp:1427233685486030067 Hops:0 NodeAddr:<nil> peerAddr:<nil> seq:0}
=== RUN TestGroups100Keys
--- PASS: TestGroups100Keys (0.10s)
=== RUN TestSameKeySameTimestamp
--- PASS: TestSameKeySameTimestamp (0.00s)
=== RUN TestSameKeyDifferentHops
--- PASS: TestSameKeyDifferentHops (0.00s)
=== RUN TestGroupGetInfo
--- PASS: TestGroupGetInfo (0.00s)
--
--- PASS: TestConvergence (0.44s)
=== RUN TestGossipInfoStore
--- PASS: TestGossipInfoStore (0.00s)
=== RUN TestGossipGroupsInfoStore
--- PASS: TestGossipGroupsInfoStore (0.00s)
FAIL
FAIL	github.com/cockroachdb/cockroach/gossip	0.713s
?   	github.com/cockroachdb/cockroach/gossip/simulation	[no test files]
=== RUN TestGetFirstRangeDescriptor
I0324 21:47:09.861223     193 network.go:62] simulating gossip network with 3 nodes
I0324 21:47:09.866158     193 server.go:93] serving on /tmp/unix-socket166524144...
I0324 21:47:09.866265     193 server.go:93] serving on /tmp/unix-socket310313871...
I0324 21:47:09.866339     193 server.go:93] serving on /tmp/unix-socket558111650...
I0324 21:47:09.866481     193 gossip.go:373] bootstrapping gossip protocol using host /tmp/unix-socket558111650
I0324 21:47:09.866879     193 gossip.go:373] bootstrapping gossip protocol using host /tmp/unix-socket558111650
I0324 21:47:09.867179     193 gossip.go:373] bootstrapping gossip protocol using host /tmp/unix-socket166524144
I0324 21:47:09.873412     193 client.go:142] client /tmp/unix-socket558111650 connected

```
Please assign, take a look and update the issue accordingly.

--- a/gossip/group_test.go
+++ b/gossip/group_test.go
@@ -172,12 +172,12 @@ func TestGroupCompactAfterTTL(t *testing.T) {
 
 	// First two inserts work fine.
 	info1 := newTestInfo("a.a", int64(1))
-	info1.TTLStamp = info1.Timestamp + int64(time.Millisecond)
+	info1.TTLStamp = info1.Timestamp + int64(5*time.Millisecond)
 	if _, err := group.addInfo(info1); err != nil {
 		t.Error(err)
 	}
 	info2 := newTestInfo("a.b", int64(2))
-	info2.TTLStamp = info2.Timestamp + int64(time.Millisecond)
+	info2.TTLStamp = info2.Timestamp + int64(5*time.Millisecond)
 	if _, err := group.addInfo(info2); err != nil {
 		t.Error(err)
 	}
@@ -188,8 +188,8 @@ func TestGroupCompactAfterTTL(t *testing.T) {
 		t.Error("shouldn't be able to insert")
 	}
 
-	// Now, wait a millisecond and try again.
-	time.Sleep(time.Millisecond)
+	// Now, wait for the TTL and try again.
+	time.Sleep(5 * time.Millisecond)
 	if _, err := group.addInfo(info3); err != nil {
 		t.Error(err)
 	}

